index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="mylogo">
+                <h1>
+                    My Logo
+                </h1>
+            </div>
+            <div class="options"> 
+                <a href="#">
+                    <img src="img/Union.png" alt="Union">
+                </a>
+            </div>
+        </nav>       
+    </header>
+    <section class="home">
+        <div class="middle">
+            <div class="text1">
+                <h2>I'm Shakespear Glad to see you!</h2>
+            </div>
+            <div class="text2">
+                <p>
+                    I am an Information and Communication Engineer and full stack website developer. I can help you build a product, feature or website look throug some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. 
+                </p>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <p>LET'S CONNECT </p>
+                </div>
+                <div class="links">
+                    <div class="tweeter"> 
```

Comment:
- [ ] Kindly use `<ul>` and `<li>` tags to list your links for accessibility reasons

- [ ] Kindly adjust the font size and margins of your design to match the Figma design.

![image](https://user-images.githubusercontent.com/98453979/198041751-0df80868-f119-496c-974f-74d0865073c5.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/198042018-93a48e8e-7a9c-4a60-9308-74a88ef2241c.png)

Reply:
## Resolved :+1
- adjusted the fonts error
- used Gogle fonts to import multiple weights
- weight include
    - 400
    - 200
    - 100
- thanks for noticing

Reply:
Resolved 👌
- changed from using `div` to `ul` and `li`
- thanks for the assist